modules/cnvpytor/importreaddepth/meta.yml

```diff
@@ -18,14 +18,23 @@ input:
       description: |
         Groovy Map containing sample information
         e.g. [ id:'test' ]
-  - bam:
+  - input_file:
       type: file
       description: BAM/CRAM/SAM file
-      pattern: "*.{bam,cram,sam}"
-  - bai:
+      pattern: "*.{bam,cram}"
+  - index:
       type: file
       description: bam file index
-      pattern: "*.{bai}"
+      pattern: "*.{bai,crai}"
+  - fasta:
+      type: file
+      description: specifies reference genome file (only for cram file without reference genome)
+      pattern: "*.{fasta,fa,fa.gz}"
```

Comment:
fasta.gz?

Reply:
Adressed in 63467f16f27ab66a1922804b8babbcfac0c0bc00